EIPS/eip-5252.md

```diff
@@ -0,0 +1,237 @@
+**`Finance`** contract, its **`Account-bound token`** can show how much balance is in agreement using SVG.
+
+**`Extension`**: **`Extension`** contract is another contract that can utilize locked funds in **`Finance`** contract. The contract can access with **`Finance`** contract on operator's approval managed in **`Manager`** contract. Example use case of `Extension` can be a membership.
+
+**`Metadata`**: **`Metadata`** contract is the contract where it stores metadata related to account credentials. Credential related data are stored with specific key. Images are usually displayed as SVG, but offchain image is possible. 
+
+---
+
+### Governance
+
+The governance pattern consists of 2 components; influencer and governor. 
+
+#### Interfaces
+
+#### Requirements
+
+An influencer contract is defined with these properties:
+1. The contract SHALL manage multiplier for votes.
+2. The contract SHALL set a decimal to calculated normalized scores.
+3. The contract SHALL set a function where governance can decide factor parameters.
+
+A governor contract is defined with these properties:
+1. The contract MUST satisfy Governor contract from OpenZeppelin.
+2. The contract SHALL refer influencer contract for multiplier
+3. The contract MUST limit transfer of account bound token once claimed for double vote prevention. 
+
+#### From Token Governance To Contribution Based Governance
+
+|               | Token Governance                 | Credential-based Governance                      |
+|---------------|----------------------------------|--------------------------------------------------|
+| Enforcement   | More tokens, more power          | More contribution, More power                    |
+| Incentives    | More tokens, more incentives     | More contribution, more incentives           |
+| Penalty       | No penalty                       | Loss of power                  |
+| Assignment | One who holds the token | One who has the most influence |
+
+<figcaption align = "center"><b>Token Governance vs Credential Based Governance</b></figcaption>
+
+
+Token governance is not sustainable in that it gives **more** power to "those who most want to rule". Any individual who gets more than 51% of token supply can change the rules to their favor.
+
+For example, Terraform Labs kept approximately 70% of their tokens or they printed more if the founder wanted to. They could manipulate voting turnout and decisions. 
+
+New governance that considers contributions to the protocol is needed because:
+
+- **Rulers can be penalized on breaking the protocol**
+- **Rulers can be more effectively incentivized on maintaining the protocol**
+
+The power should be given to "those who are most responsible". Instead of locked or owned tokens, voting power is determined with contributions are marked in `Account Bound Token`(ABT). This EIP defines new voting power as **`Influence`**.
```

Comment:
```suggestion
The power should be given to "those who are most responsible". Instead of locked or owned tokens, voting power is determined with contributions marked in Account Bound Tokens (ABT). This EIP defines this form of voting power as **`Influence`**.
```
Recommended: Rephrasing
Recommended: Formatting